controllers/operands/kubevirt.go

```diff
@@ -502,6 +502,10 @@ func getKVConfig(hc *hcov1beta1.HyperConverged) (*kubevirtcorev1.KubeVirtConfigu
 		config.Instancetype = hc.Spec.InstancetypeConfig.DeepCopy()
 	}
 
+	if hc.Spec.CommonInstancetypesDeployment != nil {
+		config.CommonInstancetypesDeployment = hc.Spec.CommonInstancetypesDeployment.DeepCopy()
```

Comment:
I wonder if we need to set the default value here. We d3fine default value for most of th th fields. @tiraboschi wdyt?

Reply:
I'm not sure I follow, kubevirt itself handles defaults for this struct when it is nil so HCO shouldn't need to do anything here?

Reply:
I tend to agree. But please see the discussion in issue #3163